src/routes/newsletter/subscribe/+page.server.ts

```diff
@@ -1,5 +1,5 @@
 import { invalid } from '@sveltejs/kit';
-import type { Actions } from './$types';
+import type { Actions } from '../$types';
```

Comment:
Accidental? This should be reverted to `./`

Reply:
yep, accidental